repository.rosinstall

```diff
@@ -17,3 +17,8 @@
     local-name: common_resources
     uri: https://github.com/shadow-robot/common_resources.git
     version: kinetic-devel
+    
```

Comment:
This is bad for a public repo, we need to fix that as soon as possible before making every package dependent on sr_manipulation